xmppserver/src/main/webapp/offline-messages.jsp

```diff
@@ -161,6 +173,14 @@
         if (quota < 0) {
             quota = 0;
         }
+
+        offlinecleaner=manager.getAutoCleanOfflineMessages()?"true":"";
+        offlinechecktimer=String.valueOf(manager.getAutoCleanOfflineMessagesTimer());
```

Comment:
I don't think you're required to use Strings - you could've just used then numeric values.

Reply:
ok next time ;)